sagemaker-geospatial/sentinel1-insar-snap/sentinel1_insar_kumamoto.ipynb

```diff
@@ -0,0 +1,1255 @@
+{
```

Comment:
<p>Is this data legally approved for use?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>